spec/support/current_academic_cycle.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+def current_academic_year
+  Time.zone.now.month > 8 ? Time.zone.now.year : Time.zone.now.year - 1
+end
+
+AcademicCycle.class_eval do
```

Comment:
Not ideal, but it's the only way to ensure we get the current academic record since the real method uses `Time.zone.now`.

Wondering if we should use `Timecop` instead to set the year during spec run?